redlines/redlines.py

```diff
@@ -133,22 +133,43 @@ def opcodes(self) -> list[tuple[str, int, int, int, int]]:
     def output_markdown(self) -> str:
         """Returns the delta in Markdown format."""
         result = []
-        style = "red"
+        default_style = 'red_green'
+        style = default_style
+
+        elem_attributes = {
+            "ins": { "class": "inserted" },
+            "del": { "class": "deleted" },
+        }
 
         if self.options.get("markdown_style"):
             style = self.options["markdown_style"]
 
         if style == "none":
             md_styles = {"ins": ("ins", "ins"), "del": ("del", "del")}
-        elif "red":
+        elif style == 'red_green':
+            md_styles = {
+                "ins": (
+                    f"span {elem_attributes['ins']['class']} style='color:green;font-weight:700;'", 
```

Comment:
should here be f"span class='{elem_attributes['ins']['class']}' style='color:green;font-weight:700;'", ?